package.json

```diff
@@ -24,18 +24,20 @@
   },
   "homepage": "https://github.com/AmpersandJS/ampersand-paginated-subcollection",
   "devDependencies": {
-    "ampersand-collection": "^1.4.0",
+    "ampersand-collection": "^1.4.5",
     "jshint": "^2.6.0",
-    "precommit-hook": "^1.0.7",
+    "precommit-hook": "^2.0.1",
     "run-browser": "^2.0.1",
-    "tap-spec": "^2.2.0",
+    "tap-spec": "^3.0.0",
     "tape": "^3.4.0"
   },
+  "files": [
+    "ampersand-paginated-subcollection.js"
```

Comment:
haven't seen this before, does it mean "only" this file is included? If so, wouldn't we want the readme too?

https://docs.npmjs.com/files/package.json#files